trace-test-application/src/androidTest/java/io/bitrise/trace/testapp/screen/IndexActivityScreen.java

```diff
@@ -34,7 +31,16 @@ public IndexActivityScreen(@NonNull final UiDevice uiDevice) {
      */
     @NonNull
     public MainActivityScreen launchUiTests() {
-        click(uiTestsButton);
+        UiObject button = uiDevice.findObject(new UiSelector()
```

Comment:
the change here is i followed the documentation to access ui components - https://developer.android.com/training/testing/ui-testing/uiautomator-testing#accessing-ui-components - i could refactor this out but it's only used in two places? What do you think?

Reply:
Something is not ok with it, as now the CI fails on all API levels. Could you please check it?

Reply:
Or, as Colin approved it, you can revert, suppress the running below API level 22, and if the CI passes we can merge it.

Reply:
I'm aware it failed - but it failed on the next part which demonstrated that getting element by text does work - so i was trying to update all the "click" to pass a UiSelector, not a BySelector. I expect it to fail a few more times until i catch all of them.

Reply:
So, I think i was going down the right track with getting elements by text, nor resource id - but it'd involve changing too much for this PR. I'm not wholly confident that this change actually worked either - the tests passed but the screenshots did not so i think it was a false positive (see the test-sdk21 workflow build) :(

I've created https://bitrise.atlassian.net/browse/APM-2795 to track the work needed to fix - we can discuss there the proposed way forward?

Reply:
Sure, let's discuss it in the mentioned ticket.